hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -898,15 +898,16 @@
             range clock option via a flag, and running several existing chip-level tests with that
             option, e.g.
```

Comment:
"running several exsiting tests" feels a little vague. Instead, could we clarify what functionalities are expected to run with reduced clock freq? Perhaps amend this to say:
```
  Test the following functionalities with reduced clock:

  - flash_ctrl initialization
  - flash_ctrl program, read and erase operations
  - AES, HMAC, KMAC and OTBN operations
  - Keymgr key derivation
  - Scramble-enabled access from the main SRAM
```

That way, we can better audit and ensure that this list (which should be moved to the `tests: []` key) is sufficient and complete.